### PR TITLE
PT-844 integrate changes for new infusion site picto

### PR DESCRIPTION
**Pull request by franck-fourel**

### CODE REVIEW

**Comment by franck-fourel** [comment]
here is the fix that I apply for the docker build failure https://stackoverflow.com/questions/52196518/could-not-get-uid-gid-when-building-node-docker

**Comment by franck-fourel** [comment]
still using tideline 1.4.0-rc1

**Comment by franck-fourel** [comment]
Adding reference to viz 0.7.0-rc1 that is the output of PR https://github.com/mdblp/viz/pull/36

File: locales/fr/translation.json [hunk 1]
```diff
@@ -203,14 +204,15 @@
...
   "Date of diagnosis: {{date}}": "Date du diagnostic : {{date}}",
-  "Date range: {{dateRange}}": "Plage de dates : {{dateRange}}",
+  "Date range: ": "Plage de dates : ",
   "day": "jour",
   "Day": "Jour",
   "day_timeago": "",
   "days": "jours",
   "Days In Report": "Jours dans le rapport",
   "days_timeago": "",
   "ddd": "",
```
Comment (yvon-dblg) [inline locales/fr/translation.json]: Here the missing translation

**Pull request merged**